catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/ResourceOperations.java

```diff
@@ -665,11 +663,24 @@ protected ResourceInfo getResourceInfo(ResourceRequest resourceRequest, String s
     }
 
     private Query anyTag(Query query) {
+        //Any metacard tag
```

Comment:
Need to do some kind of check to to see if this is a local query. If it is not then the tag filters shouldn't be applied.